### PR TITLE
[Backport fourth] Update README.md

### PR DESCRIPTION
**Pull request by ewanharris**
Backport of #48.
See that PR for full details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 samir you're not breaking the car
 
-test 22
+test 23
```
Comment (alasdairhurst) [inline README.md:3]: 23 is outdated, should be 24 now.

**Review by alasdairhurst — CHANGES_REQUESTED** [review]

**Comment by alasdairhurst** [comment]
Please fix

**Comment by ewanharris** [comment]
@alasdairhurst please fork the project if you need this fix earlier

**Comment by alasdairhurst** [comment]
Sorry,  we need this particular repository otherwise our process breaks. It's not possible to use a fork

**Pull request closed without merging**